website/content/en/docs/b1.ansible_concepts.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Understanding Ansible Concepts"
+weight: 21 # Assuming 'b' series starts after 'a' series (a1, a2), adjust if needed
+---
+
+This document provides an overview of core Ansible concepts and how they are utilized in the `infra-bootstrap-tools` project. Understanding these will help you navigate and customize the automation scripts.
+
+## 1. Playbooks
+
+*   **What they are:** Ansible Playbooks are YAML files that define a set of tasks to be executed on managed nodes (servers). They are the core of Ansible's configuration management and orchestration capabilities.
+*   **In this project:**
+    *   The main playbook is `ansible/playbooks/main.yml`. This orchestrates the entire infrastructure setup, from provisioning with Terraform to deploying applications like Caddy and Portainer.
+    *   Other playbooks like `ansible/playbooks/terraform.yml` might handle specific, smaller parts of the process.
```

Comment:
```suggestion
    *   Other playbooks like `ansible/playbooks/terraform.yml` handle specific, smaller parts of the process, so that we can apply only part of the setup or iterate faster when testing a specific module
```